### PR TITLE
New Design Discs

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
adds many new design discs, including two new categories of old NT and old syndicate plus some other missing ones for other factions. total list of added discs
Old NT: MK58, Valkyrie, Regulator, Destiny, Counselor
Old Syndicate: C20-r, Clarissa
Oberth: STS, Shellshock
Moebius: C20-m, Anne
28-Apr-22 11-29-33

## Why It's Good For The Game
adds a lot more variety to the weapon discs found in maintenance, plus QOL for neotheo

## Changelog
🆑
add: new weapon discs for Moebius, Oberth, old NT and Syndicate
imageadd: old NT and Syndicate data discs
tweak: neotheo energy weapon discs now NT cells
balance: neotheo cells and pouches no longer have license points
/🆑

### CODE REVIEW

**Comment by Humonitarian** [comment]
Last I checked Clarissa is Frozen Star? Does this PR change that?

**Comment by SirRichardFrancis** [comment]
> Last I checked Clarissa is Frozen Star? Does this PR change that?

image

File: code/game/objects/items/weapons/design_disks/oberth_republic.dm [hunk 1]
```diff
@@ -21,4 +21,28 @@
...
+	disk_name = "Oberth Republic - .30 STS Assault Rifle"
+	icon_state = "black"
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED
+	rarity_value = 65
+	license = 12
+	designs = list(
+		/datum/design/autolathe/gun/sts35 = 3, // "OR STS"
+		/datum/design/autolathe/ammo/lrifle,
+		/datum/design/autolathe/ammo/lrifle/practice = 0,
+		/datum/design/autolathe/ammo/lrifle/rubber,
```
Comment (Humonitarian) [inline code/game/objects/items/weapons/design_disks/oberth_republic.dm:36]: I really don't see why assault rifles should have rubbers in the design disk, especially alongside lethals.

File: code/game/objects/items/weapons/design_disks/syndicate.dm [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Syndicate
+/obj/item/computer_hardware/hard_drive/portable/design/syndicate
+	disk_name = "Hanza Syndicate"
+	icon_state = "syndicate"
+	license = -1
+
+/obj/item/computer_hardware/hard_drive/portable/design/guns/c20r
+	disk_name = "Hanza Syndicate - .35 C20-r submachine gun"
```
Comment (Humonitarian) [inline code/game/objects/items/weapons/design_disks/syndicate.dm:8]: What makes it Hanza? Isn't this old syndicate?

File: code/game/objects/items/weapons/design_disks/oberth_republic.dm (hunk 1 above)
Comment (Iglpigl) [reply code/game/objects/items/weapons/design_disks/oberth_republic.dm:36]: Cry about it.

**Review by Firefox13 — COMMENTED** [review]
I understand the clear intention behind this PR but I do not agree with bloating the disks spawners with new ones with rare (like Destiny, for fuck's sakes) weapons - this is because some of them are meant to be rare (like STS or, again, Destiny) and since you can hack disks this can lead to some *interesting* results.
If a contractor wants to steal Destiny for sweet telecrystals they should either get lucky with a stash or steal captain's one, not make their own.

**Comment by Firefox13** [comment]
I also do not like NT disks, Nanotrasen is like, gone, with most (if not everything) old gear having Neotheology sticker on them.
We can write it off as "there were no modern autolathe disks during corporate wars"

**Pull request closed without merging**